### PR TITLE
Adjust to multi-board support in common pkgs

### PR DESCRIPTION
**Pull request by mmoskal**
This adjusts to https://github.com/Microsoft/pxt-common-packages/pull/140

Most changes are in generated files. Real changes are:
* simulator now uses hardware IDs to lookup pins and touch pins (though not buttons) to match C++
* the pinout is defined in `config.ts` and `device.d.ts`
* `pxt.json` files are simplified
* `devpins.h/cpp` are out

I'm not sure if you want to wait with that until after the CPX release?

### CODE REVIEW

**Comment by pelikhan** [comment]
We recommend to not wait.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by pelikhan** [comment]
Still a bunch of build issues:
```
building docs in libs/music
test.ts(1,6): error TS2339: Property 'D13' does not exist on type 'typeof pins'.
test.ts(7,10): error TS2339: Property 'D13' does not exist on type 'typeof pins'.
test.ts(12,10): error TS2339: Property 'D13' does not exist on type 'typeof pins'.
test.ts(1,1): error TS9235: Unknown or undeclared identifier
test.ts(1,1): error TS9201: unsupported type: any 0x1
test.ts(1,1): error TS9201: unsupported type: any 0x1
test.ts(7,5): error TS9235: Unknown or undeclared identifier
test.ts(7,5): error TS9201: unsupported type: any 0x1
test.ts(7,5): error TS9201: unsupported type: any 0x1
test.ts(12,5): error TS9235: Unknown or undeclared identifier
test.ts(12,5): error TS9201: unsupported type: any 0x1
test.ts(12,5): error TS9201: unsupported type: any 0x1
Package built; written to binary.uf2; size: 0
```

File: docs/tutorials.md [hunk 1]
```diff
@@ -15,7 +15,6 @@ Step by step guide to coding your @boardname@.
 }, {
   "name": "Siren", 
   "cardType": "tutorial",
-  "url":"/tutorials/siren",hid
-  "imageUrl":"/static/cp/projects/siren.jpg"
+  "url":"/tutorials/siren"
```
Comment (mmoskal) [inline docs/tutorials.md]: This is what happens when I try to do 4 things at once ;)

Comment (pelikhan) [reply docs/tutorials.md]: I'll try 6 at once :)

**Comment by pelikhan** [comment]
Needs https://github.com/Microsoft/pxt-common-packages/pull/143

**Pull request merged**